### PR TITLE
Give defaults to old incomplete form sessions, sort on server side

### PR DESCRIPTION
**Pull request by wpride**
Fixes https://github.com/dimagi/commcare-hq/pull/12913

Cross https://github.com/dimagi/commcare-hq/pull/12913

@benrudolph I'm not too worried about the default values since these should only ever show up for people on the test versions

### CODE REVIEW

File: src/main/java/db/migration/V4__incomplete_forms_default.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * @author wspride
+ */
+public class V4__incomplete_forms_default implements SpringJdbcMigration {
+
+    @Override
+    public void migrate(JdbcTemplate jdbcTemplate) throws Exception {
+        jdbcTemplate.execute("ALTER TABLE formplayer_sessions " +
+                "ALTER title SET DEFAULT 'CommCare Form'");
+        jdbcTemplate.execute("ALTER TABLE formplayer_sessions " +
+                "ALTER dateOpened SET DEFAULT 'Tue Jul 3 14:19:24 EDT 1990'");
```
Comment (benrudolph) [inline src/main/java/db/migration/V4__incomplete_forms_default.java]: this is kind of a confusing default, can we set it to either `CURRENT_DATE` or Jan 1 1970 (beginning of epoch)?

File: src/main/java/db/migration/V5__incomplete_forms_set.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * Give old incomplete forms a default value
+ */
+public class V5__incomplete_forms_set implements SpringJdbcMigration {
+    @Override
+    public void migrate(JdbcTemplate jdbcTemplate) throws Exception {
+        jdbcTemplate.execute("UPDATE formplayer_sessions " +
+                "SET title='CommCare Form' WHERE title IS NULL");
+        jdbcTemplate.execute("UPDATE formplayer_sessions " +
+                "SET dateOpened='Tue Jul 3 14:19:24 EDT 1990' WHERE dateOpened IS NULL");
```
Comment (benrudolph) [inline src/main/java/db/migration/V5__incomplete_forms_set.java]: same as above

File: src/main/java/db/migration/V4__incomplete_forms_default.java (hunk 1 above)
Comment (wpride) [inline src/main/java/db/migration/V4__incomplete_forms_default.java]: Ben what is confusing about this its my birthday

**Comment by benrudolph** [comment]
not too worried about the default values either, but still nice to have something logical so that it's fairly obvious that the default value was used when we inspect the data

**Pull request merged**